customizepkg

```diff
@@ -98,6 +98,14 @@ modify_file()
 					sed -i "/^pkgname/i${context}=(${value})" "${scriptfile}"
 				fi
 				;;
+			addline)
+				echo "=> add whole line '${value}' after ${pattern} in ${context}"
+				if [ "${context}" = "global" ]; then
+					sed -i "/${pattern}/a${value}g" "${scriptfile}"
```

Comment:
So when someone wants to add a line, it should only happen once, just after the first match?